release/2.15 (#3266)
PnX-SI/GeoNature: docs/CHANGELOG.md

- [Synthèse] Possibilité de partager une URL de redirection vers un onglet (détails, taxonomie, discussion, validation, etc.) de la fiche d'une observation (#3169)
- [Accueil] Ajout d'un bloc `Discussions` sur la page d'accueil, désactivable avec le paramètre `DISPLAY_LATEST_DISCUSSIONS` (#3138)
  - Filtrable sur uniquement les discussions dans lesquels l'utilisateur authentifié a participé, ou associé à une des observations dont il est : soit l'observateur ou l'opérateur de la saisie (#3194)
- [Occhab] Remplacement du champ `is_habitat_complex` par le nouveau champ `id_nomenclature_type_habitat` et intégration de la nomenclature SINP associée (voir MosaiqueValue dans la version 2 du standard SINP) (#3125)
- [Occhab] Affichage de l'UUID de la station dans sa fiche détail (#3247)
- [Occhab] Amélioration de l'export des données en revoyant la vue `pr_occhab.v_export_sinp` (#3122)
- [Métadonnées] Possibilité de supprimer un cadre d'acquisition vide (#1673)
- [Occtax] Ajout du nom de lieu dans le détail d'un relevé (#3145)
- [RefGeo] De nouvelles mailles INPN sur la France métropolitaine (2km, 20km, 50km) sont disponibles (https://github.com/PnX-SI/RefGeo/releases/tag/1.5.4)

> - [Monitoring] Ajout de la gestion de certaines tables directement dans GeoNature (#2824)

Reviewer: ```suggestion
- [Monitoring] Ajout de la gestion de la table `gn_monitoring.t_observations` directement dans GeoNature (#2824)
```

- Correction d'un soucis de duplication des géométries quand on modifie un polygone (#3195)
- Correction de la recherche avancée par zonage dans le module Métadonnées (#3250)
- Correction d'un scroll non souhaité dans l'interface du module Synthèse (#3233)
- Correction de l'affichage des acteurs dans les fiches des observations de la Synthèse (#3086)
- Correction du chargement des champs additionnels de type Nomenclature (#3082)
- Correction des filtres taxonomiques avancés dans le mdoule Synthèse (#3087)
- Correction de l'affichage des boutons radio quand ceux-ci sont obligatoires (#3210)
- Correction de la commande `geonature sensitivity info` (#3208)
- Correction de la redirection vers la page d'authentification quand on accède au module Admin sans être authentifié (#3171)
- Correction du scroll du menu latéral dans le module Admin (#3145)
- Corrections diverses de la documentation
- Ajout d'un action Github permettant de lancer les tests depuis des modules externes (#3232)
- Lancement de `pytest` sans _benchmark_ ne nécessite plus l'ajout de `--benchmark-skip` (#3183)

**⚠️ Notes de version**

Si vous mettez à jour GeoNature : 

> - Si vous utilisez le module Monitoring, mettez le à jour en version 0.8.0 minimum en même temps que vous mettez à jour GeoNature

Reviewer: ```suggestion
- Si vous utilisez le module Monitoring, mettez le à jour en version 1.0.0 minimum en même temps que vous mettez à jour GeoNature
```

   - Si vous n'aviez pas installé le module externe Import, il sera disponible après la mise à jour vers cette nouvelle version de GeoNature. Vous pouvez configurer les permissions de vos utilisateurs si vous souhaitez qu'ils y accédent.
   - La gestion des permissions et des JDD associés aux module a évolué. La migration est gérée automatiquement lors de la mise à jour pour garantir un fonctionnement identique.
   - Reporter l'éventuelle configuration de votre module Import dans le fichier de configuration de GeoNature (dans le bloc import, voir dans le fichier default toml)
- La synchronisation avec le service MTD de l'INPN n'est plus intégrée dans le code de GeoNature, elle a été déplacée dans un module externe : https://github.com/PnX-SI/mtd_sync
   - Si vous l'utilisiez, supprimer les variables de configuration suivantes du fichier `geonature_config.toml` : 
     - `XML_NAMESPACE`, `MTD_API_ENDPOINT`
     - toutes les variables dans `[CAS_PUBLIC]`, `[CAS]`, `[CAS.CAS_USER_WS]`, `[MTD]`
     - `ID_USER_SOCLE_1` et `ID_USER_SOCLE_2` dans la section `BDD` 

> - Le champ `id_digitizer` des tables `gn_monitoring.t_base_sites`, `gn_monitoring.t_base_visits` est obligatoire. Assurez vous qu'ils soient peuplés avant la mise à jour.

Reviewer: ```suggestion
- Si vous utilisez le module Monitoring, les champs `id_digitizer` des tables `gn_monitoring.t_base_sites`, `gn_monitoring.t_base_visits` est désormais obligatoire. Assurez-vous qu'ils soient peuplés avant de lancer la mise à jour de GeoNature (`SELECT * FROM gn_monitoring.t_base_visits tbv WHERE id_digitiser IS NULL; SELECT * FROM gn_monitoring.t_base_sites tbs WHERE id_digitiser IS NULL;`).
```
